blockchain/difficulty.go

```diff
@@ -540,6 +540,34 @@ func (b *BlockChain) sumPurchasedTickets(startNode *blockNode, numToSum int64) i
 	return numPurchased
 }
 
+// LiveTicketsAtNextMaturity returns the total number of live tickets
+// available after the ticket maturity period from the provided block.
+// This assumes all blocks mined during the period are voted on
+// by the maximum number of tickets allowed per block.
+func (b *BlockChain) LiveTicketsAtNextMaturity(hash *chainhash.Hash) uint64 {
```

Comment:
Given the importance of this function, this needs a unit test to hit the edge conditions.  It can create a fake chain like several of the existing tests for this purpose since it just needs to work with nodes.